build/README.md

```diff
@@ -57,3 +48,9 @@ Build with test
 ```
 mvn clean package
 ```
+
+## For any contributors : if change format code, need build as per the command
```

Comment:
Better phrase as below.
For contributors : To build the format code after any changes, please follow the below command.